lambdas/service/app.py

```diff
@@ -825,35 +825,44 @@ def get_order():
     return {'order': Plugin.load().service_config().order_config}
 
 
-manifest_filters = {
-    facet_name: schema.object(**{'is': schema.array(str)})
-    for facet_name in Plugin.load().service_config().translation.keys()
-}
-
-manifest_param_format = params.query(
-    'format',
-    schema.optional(schema.enum('compact', 'full', 'terra.bdbag', type_=str)),
-    description='''
-        The desired format of the output.
-
-        - `compact` (the default) for a compact tab-separated manifest
-
-        - `full` for a full tab-separated manifest
-
-        - `terra.bdbag` for a manifest in the format documented
-          [http://bd2k.ini.usc.edu/tools/bdbag/]. This provides a ZIP file
-          containing two manifests: one for participants (aka Donors) and one
-          for samples (aka specimens). The format of the manifests inside the
-          BDBag is documented here:
-          [https://software.broadinstitute.org/firecloud/documentation/article?id=10954]
-    ''',
-)
-
-manifest_param_token = params.query(
-    'token',
-    schema.optional({'type': 'string'}),
-    description='Reserved. Do not pass explicitly.',
-)
+def manifest_path_spec():
```

Comment:
This works, or there could be an explicit call to `copy` when the variable is used.